src/Common/src/Interop/Windows/sspicli/SafeDeleteContext.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Authentication.ExtendedProtection;
+
+namespace System.Net.Security
+{
+    //
```

Comment:
How does this SafeDeleteContext differ from the one we already have in the repo?
[https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/Security/Unix/SafeDeleteContext.cs](https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/Security/Unix/SafeDeleteContext.cs)